src/main/java/racingcar/Car.java

```diff
@@ -1,12 +1,47 @@
 package racingcar;
 
+import static camp.nextstep.edu.missionutils.Randoms.pickNumberInRange;
 public class Car {
     private final String name;
     private int position = 0;
-
     public Car(String name) {
+
         this.name = name;
     }
+    public String getName() {
+
+        return this.name;
+    }
+
+    public void printCar() {
 
-    // 추가 기능 구현
+        System.out.println(name+" : "+getPositionString());
+    }
+
+    public void updatePosition() {
+
+        if(canAdvance()){
+            position++;
+        }
+    }
+    public boolean canAdvance() {
+
+        return pickNumberInRange(0, 9) >= 4;
+    }
```

Comment:
해당 함수도 외부에 공개될 필요가 없다면 private을 고민해 보시는것도 좋을것 같습니다.
```suggestion
    public boolean canAdvance() {

        return pickNumberInRange(0, 9) >= 4;
    }
```

Reply:
네 알겠습니다!